.github/dependabot.yml

```diff
@@ -1,8 +1,17 @@
 version: 2
 updates:
-    - package-ecosystem: 'npm'
-      directory: '/'
+    - package-ecosystem: npm
+      directory: /
       schedule:
-          interval: 'weekly'
+          interval: weekly
       open-pull-requests-limit: 5
-      versioning-strategy: 'increase'
+      versioning-strategy: increase
+      groups:
+          dev-dependencies:
+              applies-to: version-updates
+              dependency-type: development
```

Comment:
I would personally be open to / in favour of actually removing this line and do this for regular dependencies too. We already exclude security fixes and everything from `@dhis2` and I don't think we're in such a hurry with all the other stuff.